server/sonar-process/src/main/java/org/sonar/process/DefaultProcessCommands.java

```diff
@@ -52,6 +52,15 @@ public static DefaultProcessCommands secondary(File directory, int processNumber
     return new DefaultProcessCommands(directory, processNumber, false);
   }
 
+  /**
+   * Clears the shared memory space of the specified process number.
+   */
+  public static void reset(File directory, int processNumber) {
+    try (DefaultProcessCommands processCommands = new DefaultProcessCommands(directory, processNumber, true)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "processCommands" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1481)